src/SwiftReflector/Demangling/Swift4Demangler.cs

```diff
@@ -0,0 +1,1814 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Do we need a Swift 4 demangler? I though we were only going to supportd Swift 5+.

Reply:
Right, it is not needed and will be excluded from this PR.